pkgs/tools/misc/coreboot-utils/default.nix

```diff
@@ -0,0 +1,103 @@
+{ stdenv, fetchurl, zlib, pciutils, coreutils, acpica-tools, iasl, makeWrapper, gnugrep, gnused, file, buildEnv }:
+
+let
+  version = "4.10";
+
+  meta = with stdenv.lib; {
+    description = "Various coreboot-related tools";
+    homepage = "https://www.coreboot.org";
+    license = licenses.gpl2;
+    maintainers = [ maintainers.petabyteboy ];
+    platforms = platforms.linux;
+  };
+
+  generic = { pname, path ? "util/${pname}", ... }@args: stdenv.mkDerivation (rec {
+    inherit pname version meta;
+
+    src = fetchurl {
+      url = "https://coreboot.org/releases/coreboot-${version}.tar.xz";
+      sha256 = "1jsiz17afi2lqg1jv6lsl8s05w7vr7iwgg86y2qp369hcz6kcwfa";
+    };
+
+    enableParallelBuilding = true;
+
+    prePatch = ''
+      cd ${path}
+    '';
+
+    configureFlags = [
+      "INSTALL=install"
+      "PREFIX=${placeholder "out"}"
+    ];
+    installFlags = configureFlags;
+    makeFlags = configureFlags;
+  } // args);
+
+  utils = {
+    msrtool = generic {
+      pname = "msrtool";
+      meta.description = "Dump chipset-specific MSR registers";
+      buildInputs = [ pciutils zlib ];
+      preConfigure = "export INSTALL=install";
```

Comment:
Why was this needed?  (I think the configure script for this one wanted this)

Reply:
Yes, it ignores it when it comes as a parameter, but uses it when it's in the environment.